pkg/limayaml/validate.go

```diff
@@ -493,9 +515,16 @@ func validateParamIsUsed(y *limatype.LimaYAML) error {
 				keyIsUsed = true
 				break
 			}
-			if p.Content != nil && re.MatchString(*p.Content) {
-				keyIsUsed = true
-				break
+			for _, ptr := range []*string{
+				// mode: data
+				p.Content, p.Owner, p.Path,
```

Comment:
Good catch of the additional templates. Maybe add `&p.Script` as well, then you can eliminate the check above as well.